<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgTemplatesList</class>
 <widget class="QDialog" name="DlgTemplatesList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1045</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SkauTan: Templates</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Templates:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="tblTemplates">
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOn</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAsNeeded</enum>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="horizontalScrollMode">
            <enum>QAbstractItemView::ScrollPerPixel</enum>
           </property>
           <attribute name="horizontalHeaderHighlightSections">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderDefaultSectionSize">
            <number>21</number>
           </attribute>
           <attribute name="verticalHeaderMinimumSectionSize">
            <number>21</number>
           </attribute>
           <column>
            <property name="text">
             <string>Name</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>#</string>
            </property>
            <property name="toolTip">
             <string>Number of items</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Color</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Notes</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="_2">
         <item>
          <widget class="QPushButton" name="btnAddTemplate">
           <property name="text">
            <string>&amp;Add...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnRemoveTemplate">
           <property name="text">
            <string>&amp;Delete...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnMoveTemplatesUp">
           <property name="text">
            <string>Move &amp;up</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnMoveTemplatesDown">
           <property name="text">
            <string>Move do&amp;wn</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnImport">
           <property name="text">
            <string>&amp;Import...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnExport">
           <property name="text">
            <string>E&amp;xport...</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Items in the selected template:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="tblItems">
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOn</enum>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="horizontalScrollMode">
            <enum>QAbstractItemView::ScrollPerPixel</enum>
           </property>
           <attribute name="horizontalHeaderHighlightSections">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderDefaultSectionSize">
            <number>21</number>
           </attribute>
           <attribute name="verticalHeaderMinimumSectionSize">
            <number>21</number>
           </attribute>
           <column>
            <property name="text">
             <string>Name</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Notes</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Duration</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string># Songs</string>
            </property>
            <property name="toolTip">
             <string>Number of matching songs</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Filter</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="btnAddItem">
           <property name="contextMenuPolicy">
            <enum>Qt::CustomContextMenu</enum>
           </property>
           <property name="toolTip">
            <string>Insert a filter before the selected item</string>
           </property>
           <property name="text">
            <string>Insert</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnRemoveItem">
           <property name="text">
            <string>&amp;Remove...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnMoveItemUp">
           <property name="text">
            <string>Move up</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnMoveItemDown">
           <property name="text">
            <string>Move down</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnManageFilters">
           <property name="text">
            <string>&amp;Manage...</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnClose">
           <property name="text">
            <string>&amp;Close</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
